src/sublime_aio.py

```diff
@@ -124,21 +150,21 @@ def wrapper(self, *args, **kwargs) -> None:
             if __loop is None:
                 return
 
-            view = self.view if hasattr(self, 'view') else args[0]
+            view = self.view if isinstance(self, sublime_plugin.ViewEventListener) else args[0]
```

Comment:
As decorator is valid only for our own listeners...

```suggestion
            view = self.view if isinstance(self, ViewEventListener) else args[0]
```